libs/ng2-charts/src/lib/get-colors.ts

```diff
@@ -1,7 +1,8 @@
 import { baseColors } from './base-colors';
 import { Color, ScriptableContext } from "chart.js";
+import {AnyObject} from "chart.js/dist/types/basic";
 
-export const builtInDefaults = {
+export const builtInDefaults :AnyObject= {
```

Comment:
Isn't there a more restrictive type we could use for defaults in Chart.js?

Reply:
I wish, but unfortunately Chart.js typing is quite loose for that method:

https://github.com/chartjs/Chart.js/blob/767d64e7a90dbfe94f9c5d159a890054818c1680/src/types/index.d.ts#L673